### ISSUE TITLE
About the details of train/val/test split of the BCDD set

### ISSUE THREAD

**Issue by justchenhao** [lehaifeng/DASNet#5]
Greetings! Could you release the details of the train/val/test set of the BCDD set? For example, could you provide the train/val/test lists for each dataset? Because many people may want to make a fair comparison with your proposed method. Thank you very much.

**Comment by louxiantuo** [comment]
Yes, I also need the list. The list I generated by myself cannot run the code successfully. I hope the author can update it as soon as possible.
